tests/Build/Changelog/ChangelogBuilderTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Aws\Test\Build\Changelog;
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+class ChangelogBuilderTest extends \PHPUnit_Framework_TestCase
+{
+	/**
+	 * @covers ChangelogBuilder
+	 *
+	 */
+
+	private $RESOURCE_DIR= "tests/Build/Changelog/resources";
+
+	private function getChangelogBuilder()
```

Comment:
If the constructor signature changes, then the signature of this method likely would too, as you would need to test different values for those new constructor parameters. You would need to change all code that calls this function anyway.